lib/moonshot/dynamic_template.rb

```diff
@@ -24,12 +24,15 @@ def expose_binding
 
     def initialize(source:, parameters:, destination:)
       @source = File.read(source)
-      @parameters = Parameters.new(parameters)
+      @parameters = parameters
```

Comment:
We should stop accepting parameters in `initialize` if we are going to overwrite it during `process`.

Reply:
I guess we should accept parameter during initialization because in this way we are sending either lambda or array of params to the plugin.

Reply:
@pdrakeweb we should, @valentyntymku please provide an example how the plugin should be configured in the Moonfile. You might consider this to be slightly hacky, however this is a solution that doesn't require major modifications in Moonshot.

Reply:
We can use this in both ways in Moonshot.rb.
The 'old-school':
`parameters = { deletion_policy: 'Delete' }`

And the Lambda way:
`parameters = -> environment_name {
environment = environment_name =~ prod_regexp ? 'prod' : 'dev'
parameters = YAML.load_file("cloud_formation/parameters/#{environment}.yml")
}`